app/src/test/java/net/squanchy/schedule/domain/view/EventFixtures.java

```diff
@@ -1,116 +0,0 @@
-package net.squanchy.schedule.domain.view;
-
-import java.util.Collections;
-import java.util.List;
-
-import net.squanchy.eventdetails.domain.view.ExperienceLevel;
-import net.squanchy.speaker.domain.view.Speaker;
-import net.squanchy.support.lang.Optional;
-
-import org.joda.time.DateTimeZone;
-import org.joda.time.LocalDateTime;
-
-import static net.squanchy.schedule.domain.view.PlaceFixtures.aPlace;
-import static net.squanchy.schedule.domain.view.TrackFixtures.aTrack;
-
-public final class EventFixtures {
```

Comment:
This (and the other fixtures from the package) have been moved into `ScheduleFixtures.kt` in the same package